internal/spokes/spokes.go

```diff
@@ -79,6 +79,7 @@ func Exec(ctx context.Context, stdin io.Reader, stdout io.Writer, stderr io.Writ
 		return 1, err
 	}
 
+	quarantineFolder := filepath.Join(repoPath, "objects", quarantineID)
```

Comment:
Can this be inlined again, now that `rp.Close()` takes care of cleaning up the quarantine on error?